adapter/config/default_config.go

```diff
@@ -91,6 +91,18 @@ var defaultConfig = &Config{
 				DisableSSLVerification: false,
 			},
 		},
+		TimeOuts: timeOuts{
+			Route: route{
+				RouteTimeoutInSeconds:     60,
+				RouteIdleTimeoutInSeconds: 300,
+			},
+			Connection: connection{
+				RequestTimeoutInSeconds:        0,
+				RequestHeadersTimeoutInSeconds: 0,
```

Comment:
Is default 0 values are correct for these configs

Reply:
As per the envoy docs, if not specified or set to 0, these timeouts are disabled. 

So IMO we can set them to 0. wdyt?

https://github.com/envoyproxy/envoy/blob/0e26d7fb01d1ebfe481e5fccd91d3e2e87030b9f/api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#L422-L435

Reply:
+1